bitnami/ejbca/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: ejbca
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/ejbca
-version: 14.0.3
+version: 14.0.4
```

Let's bump the chart version again
```suggestion
version: 14.0.5
```